### PR TITLE
Improve documentation

### LINKED ISSUE #73: [User Story]: Clear Separation between Readme and Contributing file
### Description

As a maintainer I want a readme which explains the setup which potentially references a setup file for more details. The contributing file should not contain any setup contents.

### Acceptance Criteria

The Readme and a potential setup file contain setup and other descriptions, the contributing guide should not contain any of these. 

### Comment

For the contributing guide file see for example here: https://contributing.md/example/

### LINKED ISSUE #152: [User Story]: Improve Test documentation
### Description

As a developer I want a better documentation on how to perform tests.

### Acceptance Criteria

The readme gives info about how to run the different tests and how to run with coverage. 

### Comment

_No response_

### PR DESCRIPTION
**Pull request by niklhut**
Fixes #73 and fixes #152

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,50 @@
...
+npm ci
 ```
+
+Only use `npm install` if you want to update the dependencies or add new ones.
+
+### Environment Variables
+
+Create a `.env`, `.env.test` and `.env.development` file in the root directory of the project. Fill them with the values of the `.env.example` file.
+
+In the `.env` file change `localhost` in the `DATABASE_URL` to `db` for the docker container to work correctly.
```
Comment (Tim-Kroker) [inline README.md:17]: I think this line comes from a time when we used sequelize

**Review by Tim-Kroker — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Tim-Kroker) [reply README.md:17]: Line 17

Comment (niklhut) [reply README.md:17]: No this is up to date, in the .env we have this variable `DATABASE_URL=postgresql://nuxt:nuxt_password@localhost:5432/phyexdb` and in it localhost should be changed to db in the .env file, for the other envs localhost is correct

**Review by Tim-Kroker — APPROVED** [review]
lgtm

**Pull request merged**